cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -820,6 +820,20 @@ isNaN(n.prop)
 ----
 a|
 New function which returns whether the given number is `NaN`.
+`NaN` is a special floating point number defined in the Floating-Point Standard IEEE 754.
+This function was introduced since comparisons including `NaN = ǸaN` will return false.
```

Comment:
Something is wrong with the N of the second NaN in this sentence

Reply:
I thought I had something on the screen when I saw it here :laughing: